packages/client/styles/paletteV3.ts

```diff
@@ -4,79 +4,84 @@
 //       We try to only use the limited palette, but weighted variants
 //       can help with subtle color changes in states like hover, or disabled.
 //       The limited palette is here:
-//       https://www.figma.com/file/OA9NkpSTlHVqqRL9IE9KsF/Palette-v3?node-id=15%3A366
+//       https://www.figma.com/file/OA9NkpSTlHVqqRL9IE9KsF/Palette-v3?node-id:15%3A366
 //       The expanded palette is here:
-//       https://www.figma.com/file/OA9NkpSTlHVqqRL9IE9KsF/Palette-v3?node-id=15%3A2184
+//       https://www.figma.com/file/OA9NkpSTlHVqqRL9IE9KsF/Palette-v3?node-id:15%3A2184
 
-export const enum PALETTE {
-  AQUA_400 = '#55C0CF',
+import tailwindPreset from '../tailwind.preset.js'
 
-  FOREST_400 = '#73A153',
+const {colors} = tailwindPreset.theme
 
-  FUSCIA_400 = '#DB70DB',
+export const PALETTE = {
```

Comment:
It can't be an enum anymore as TS complains we can't use computed values in enums